data.sql

```diff
@@ -40,4 +40,57 @@ UPDATE animals SET species_id =
     WHEN name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom' THEN 4
     WHEN name = 'Angemon' OR name = 'Boarmon' THEN 5
   END;
-  
\ No newline at end of file
+
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specialties (vet_id, species) VALUES
+(1, 'Pokemon'),
+(3, 'Digimon'),
+(3, 'Pokemon'),
+(4, 'Digimon');
+
+-- Insert data for vets
+INSERT INTO vets (name, age, graduation_date)
+VALUES ('William Tatcher', 45, '2000-04-23'),
+       ('Maisy Smith', 26, '2019-01-17'),
+       ('Stephanie Mendez', 64, '1981-05-04'),
+       ('Jack Harkness', 38, '2008-06-08');
+
+-- Insert data for specializations
+INSERT INTO specializations (vet_id, species)
+VALUES (1, 'Pokemon'),
+       (3, 'Digimon'),
+       (3, 'Pokemon'),
+       (4, 'Digimon');
```

Comment:
[OPTIONAL]
- [ ] Instead of hard-coded the `id` of your vets it would be nice and efficient to get them dynamically like for example:
```sql
INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));
```